next/components/forms/SelectField/SelectFieldBox.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { ForwardedRef, forwardRef, ForwardRefRenderFunction } from 'react'
+
+import Tag from '../Tag'
+import SelectOption from './SelectOption'
+
+interface SelectFieldBoxProps {
+  value?: SelectOption[]
+  multiple?: boolean
+  placeholder?: string
+  filter: string
+  filterRef?: React.RefObject<HTMLInputElement>
+  onRemove: (optionId: number) => void
+  onFilterChange: (value: string) => void
+  onFilterFocusChange: (isFocused: boolean) => void
+  onDeleteLastValue: () => void
+}
+
+const SelectFieldBoxComponent: ForwardRefRenderFunction<HTMLDivElement, SelectFieldBoxProps>
+  = (props: SelectFieldBoxProps, ref:ForwardedRef<HTMLDivElement>) => {
+  // PROPS
+  const { value, multiple, placeholder, filter, filterRef, onRemove, onFilterChange, onFilterFocusChange, onDeleteLastValue } = props
+
+  // HELPER FUNCTIONS
+  const getInputSize = () => {
+    return !value || value.length === 0
+      ? 13
```

Comment:
this works well enough, but may cause friction with our tester unless it's specifically agreed upon with design already

no action required as of now, just fyi